packages/visual-dev/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/visual-dev",
-  "version": "1.4.0",
+  "version": "1.5.0-3",
```

Comment:
Need to remember to add git tags for this new version at the bottom of the CHANGELOG